doc/api/fs.md

```diff
@@ -1147,6 +1147,33 @@ platform-specific. On macOS, Linux, and Windows, the promise will be rejected
 with an error. On FreeBSD, a representation of the directory's contents will be
 returned.
 
+An example of reading a `package.json` file located at `/home/user/project/package.json`:
+```mjs
+import { readFile } from 'fs/promises';
+
+// Where the value of `filename` is the name of the file you want to read
+const filename = '/home/user/project/package.json';
+
+const contents = await readFile((filename), { encoding: 'utf8' });
+
+console.log(contents);
+```
+
+```cjs
+const { readFile } = require('fs/promises');
+
+async function logFile() {
+  // Where the value of `filename` is the name of the file you want to read
+  const filename = '/home/user/project/package.json';
+
+  const contents = await readFile((filename), { encoding: 'utf8' });
```

Comment:
```suggestion
  const contents = await readFile(path.join(__dirname, 'package.json'), { encoding: 'utf8' });
```

Reply:
I intentionally wrote it so that it wasn't going to use path. If we're going to use `path`, I'd rather do a relative path with `resolve` since it looks a bit nicer.